src/Tgstation.Server.Host/Components/Deployment/DmbFactory.cs

```diff
@@ -426,5 +423,18 @@ async Task HandleCleanup()
 					logger.LogTrace("Compile job {0} lock count now: {1}", job.Id, decremented);
 				}
 		}
+
+		/// <summary>
+		/// Handles cleaning the resources of a <see cref="CompileJob"/>.
+		/// </summary>
+		/// <param name="directory">The directory to cleanup.</param>
+		/// <param name="cancellationToken">The <see cref="CancellationToken"/> for this <see cref="Task"/>.</param>
+		/// <returns>The deletion task</returns>
```

Comment:
```suggestion
		/// <returns>The deletion <see cref="Task"/>.</returns>
```

Reply:
I floated this idea but thought it was excessive

lmao